File: 5 - Simple Math/C/main.c
#include<stdio.h>

int main(void){
    float n1;
    float n2;

    printf("What is the first number? ");
    scanf("%f", &n1);

    printf("What is the second number? ");
    scanf("%f", &n2);

    printf("%f + %f = %f\n", n1, n2, n1+n2);
    printf("%f - %f = %f\n", n1, n2, n1-n2);
    printf("%f * %f = %f\n", n1, n2, n1*n2);
    printf("%f / %f = %f\n", n1, n2, n1/n2);

    return 0;
}

File: 5 - Simple Math/Golang/main.go
package main

import "fmt"

func main() {
	var n1 float64
	fmt.Print("What is the first number? ")
	_, _ = fmt.Scanf("%64f", &n1)
	var n2 float64
	fmt.Print("What is the second number? ")
	_, _ = fmt.Scanf("%f", &n2)

	fmt.Printf("%v + %v = %v\n", n1, n2, n1+n2)
	fmt.Printf("%v - %v = %v\n", n1, n2, n1-n2)
	fmt.Printf("%v * %v = %v\n", n1, n2, n1*n2)
	fmt.Printf("%v / %v = %v\n", n1, n2, n1/n2)
}
